patches/symfony-console-helper-helper-php.patch

```diff
@@ -4,14 +4,16 @@
      public static function width(?string $string): int
      {
          $string ??= '';
-+        return \mb_strlen($string);
++        // return early, to prevent a dependency on symfony/string to ease downgrading rector.
++        return \mb_strwidth($string);
```

Comment:
we are now using `mb_strwidth` instead of `mb_strlen`, which is not needed as per conversation in https://github.com/rectorphp/vendor-patches/commit/71ce09e629e11013ac0c2b54482777a5a9fbd861#r125484668 but its more consistent and just makes more sense